aws/resource_aws_appmesh_mesh.go

```diff
@@ -157,7 +180,7 @@ func resourceAwsAppmeshMeshDelete(d *schema.ResourceData, meta interface{}) erro
 	_, err := conn.DeleteMesh(&appmesh.DeleteMeshInput{
 		MeshName: aws.String(d.Id()),
 	})
-	if isAWSErr(err, "NotFoundException", "") {
+	if isAWSErr(err, appmesh.ErrCodeNotFoundException, "") {
```

Comment:
Nice. Could you also replace the hard-coded string value in `resourceAwsAppmeshMeshRead()`? https://github.com/terraform-providers/terraform-provider-aws/blob/1bb0dead12829bf6c9da9e3170f52a8f24984497/aws/resource_aws_appmesh_mesh.go#L107

Reply:
Fixed it.